shoop/admin/modules/media/static_src/media/browser/components/folderClick.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * This file is part of Shoop.
+ *
+ * Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+ *
+ * This source code is licensed under the AGPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const _ = require("lodash");
+
+export default function(ctrl, folder) {
+    function clickFolder(event, folderId) {
+        ctrl.setFolder(folderId);
+        event.preventDefault();
+        return false;
+    }
+
+    return _.partialRight(clickFolder, folder.id);
```

Comment:
You don't need to use `_.partialRight` since you're already making `clickFolder` an inner function that has `folder` in its closure:

```javascript
export default function(ctrl, folder) {
    return function clickFolder(event) {
        ctrl.setFolder(folder.id);
        event.preventDefault();
        return false;
    }
}
```